skt/kernelbuilder.py

```diff
@@ -327,13 +336,53 @@ def make_rpm_repo(self, rpm_files):
 
         return repo_dir
 
+    def find_extras(self, rel_path, callback=None, followlinks=True):
+        """
+        Find files to be distributed alongside with the kernel.
+
+        Args:
+            rel_path:    A path (relative to the kernel tree root) in which
+                         extras are sought (recursively).
+
+            callback:    If given, serves as a result pruning function for
+                         filter.
+
+            followlinks: Follow symbolic links to subdirectories on systems
+                         that support them. Set this to True only if you're
+                         sure that this won't cause loops.
+
+        Returns:
+            A (possibly pruned) list of paths recursively found in
+            `rel_path`. Returns an empty list if no matches were found
+            or an exception occured during pruning.
+        """
+        source_dir_abs = os.path.abspath(self.source_dir)
+        target_dir = source_dir_abs + rel_path
+
+        result = []
+
+        logging.info("looking for extras in: %s", target_dir)
+        for root, _, files in os.walk(target_dir, followlinks=followlinks):
+            result += [
+                os.path.relpath(root + '/' + f, source_dir_abs)
```

Comment:
Same with the path joining here, please use `os.path.join` or `skt.misc.join_with_slash`